src/Helpers/userTypes.js

```diff
@@ -1,2 +1,10 @@
 // eslint-disable-next-line import/no-anonymous-default-export
-export default { 2: "Usuario Standard", 1: "Usuario Adminisrador" };
+ const rolesDeUsuarios = { 2: "Usuario Standard", 1: "Usuario Adminisrador" };
```

Comment:
ÉSto podría mejorarse:

1. Son constantes, deberian ir con mayuscula.
2. Agrupar ids de roles en un mismo objeto.
3. Deberia retornar 2 constantes, una con el objeto con los ids de los roles y el otro con la descripcion de cada role

Reply:
Listo. Pero tengo una duda.. cuando importo solamente la constante de ID_USUARIOS me trae las dos contstantes.. por esa razon para entrar al objeto tengo que hacerlo de la siguiente forma ID_USUARIOS.ID_USUARIOS.usuarioStandar . Como podria solucionar esto?

Reply:
Deberías hacer destructuring:
```js
import {
    usuarioStandar
} from '../../Helpers/userTypes';
```

Porque acá userTypes te está devolviendo un objet:
```js
{
    rolesDeUsuarios,
    usuarioStandard,
    usuarioAdministrador,
 }
```

Reply:
![Captura de pantalla (143)](https://user-images.githubusercontent.com/81937779/156431123-251d9577-93ec-4f67-b0b0-2b11527f9acb.png)
estuve probando hacer el destructuring pero sigue dando error

Reply:
Claro, porque estás usando export default, usa sólo export

Reply:
si, estuve intentando con export solamente y hacer el destructuring pero tampoco funciona. Me da esta error export 'default' (imported as 'userTypes') was not found in '../../Helpers/userTypes' (possible exports: ID_USERS, USER_ROLES)
Solo me funciona con export default e importando ID_USERS sin destructuring.

Reply:
Claro, pero fijate en que archivo te esta fallando, porque se usa en varios. Deberias modificar todos los archivos. Yo recién probé y funcionó